disvae/models/losses.py

```diff
@@ -359,10 +365,10 @@ def __call__(self, data, recon_batch, latent_dist, is_train, storer,
         rec_loss = _reconstruction_loss(data, recon_batch,
                                         storer=storer,
                                         distribution=self.rec_dist)
-        log_pz, log_qz, log_prod_qzi, log_q_zCx = _get_log_pz_qz_prodzi_qzCx(latent_sample,
-                                                                             latent_dist,
-                                                                             self.n_data,
-                                                                             is_mss=self.is_mss)
+        log_pz, log_qz, log_prod_qzi, log_q_zCx, _ = _get_log_pz_qz_prodzi_qzCx_qz_knc(latent_sample,
```

Comment:
should be `_get_log_pz_qz_prodzi_qzCx_qzknc` as underscores separate each outputs